sound/soc/dwc/dwc-i2s.c

```diff
@@ -281,28 +281,38 @@ static int dw_i2s_hw_params(struct snd_pcm_substream *substream,
 
 	switch (params_format(params)) {
 	case SNDRV_PCM_FORMAT_S16_LE:
-		config->data_width = 16;
-		dev->ccr = 0x00;
+		// config->data_width = 16;
+		// dev->ccr = 0x00;
+		config->data_width = 32;
```

Comment:
why?

Reply:
The source code of dwc-i2s does not make the k230 audio work properly, so it needs to be modified. Instead of directly modifying it, I will make a copy of dwc-i2s.c to enable k230.